nautobot/dcim/tests/test_forms.py

```diff
@@ -343,9 +343,20 @@ def setUpTestData(cls):
 
     def test_interface_form_clean_vlan_location_fail(self):
         """Assert that form validation fails when no matching locations are associated to tagged VLAN"""
-        self.vlan.locations.set(list(Location.objects.exclude(pk=self.device.location.pk))[:2])
+        location = self.device.location
+        location_ids = [ancestor.id for ancestor in location.ancestors()]
+        self.vlan.locations.set(list(Location.objects.exclude(pk__in=location_ids))[:2])
```

Comment:
Intentional that we're *not* excluding `self.device.location` from this list any more?